shared/components/Card.js

```diff
@@ -1,7 +1,10 @@
 import React, { PropTypes, Component } from 'react';
-import { StyleSheet, Text, View, ListView, Image, Platform, TouchableOpacity } from 'react-native';
+import { StyleSheet, Text, View, ListView, Image, Platform, TouchableOpacity, Dimensions } from 'react-native';
 import {Actions} from 'react-native-router-flux';
 
+import IMG_BUTTON_STAR from '../resources/btn_star/drawable-xxxhdpi/btn_star.png';
+import IMG_BUTTON_YELLOW_STAR from '../resources/btn_star_yellow/drawable-mdpi/btn_star.png';
+
 const FLEX_MARGIN_ROW = 16;
 const FLEX_MARGIN_TEXT_ROW = 8;
 const FLEX_LIST_WRAPPER = 120 - 36;
```

Comment:
why are you using const variable? I think if you won't define every flex value, just using constans is better.